modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/EmailTemplatesRestClient.java

```diff
@@ -0,0 +1,93 @@
+package org.wso2.identity.integration.test.restclients;
+
+import io.restassured.http.ContentType;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.wso2.carbon.automation.engine.context.beans.Tenant;
+import org.wso2.identity.integration.test.rest.api.common.RESTTestBase;
+
+import java.io.IOException;
+
+public class EmailTemplatesRestClient {
+
+    protected static final String TENANT_PATH = "t/%s";
+    protected static final String API_SERVER_BASE_PATH = "/api/server/v1";
+    public static final String EMAIL_TEMPLATES_EMAIL_BASE_PATH = "/email";
+    public static final String EMAIL_TEMPLATE_TYPES_PATH = "/template-types";
+    public static final String EMAIL_TEMPLATES_PATH = "/templates";
+    public static final String PATH_SEPARATOR = "/";
+    public static final String BASIC_AUTHORIZATION_ATTRIBUTE = "Basic ";
+    public static final String CONTENT_TYPE_ATTRIBUTE = "Content-Type";
+    public static final String AUTHORIZATION_ATTRIBUTE = "Authorization";
+    private final String EMAIL_TEMPLATE_API_BASE_PATH;
+    private final CloseableHttpClient client;
+    private final String USERNAME;
+    private final String PASSWORD;
```

Comment:
Are these constants?

Reply:
Fixed with https://github.com/wso2/product-is/pull/15931/commits/0779642105a7a7816004cc1a6befe408d3112c15